.github/workflows/openhands-resolver.yml

```diff
@@ -340,11 +359,17 @@ jobs:
               });
               process.env.AGENT_RESPONDED = 'true';
             } else if (!success && branchName) {
+              let commentBody = `An attempt was made to automatically fix this issue, but it was unsuccessful. A branch named '${branchName}' has been created with the attempted changes. You can view the branch [here](https://github.com/${context.repo.owner}/${context.repo.repo}/tree/${branchName}). Manual intervention may be required.`;
+
+              if (successExplanation) {
+                commentBody += `\n\nAdditional details about the failure:\n${successExplanation}`;
```

Comment:
Just nit, this line reads amusing. The failure is the success explanation 😅

If `successExplanation` can be both about success and failure, could we call it... idk, `attemptExplanation`?

Reply:
Or `resultExplanation`?

Reply:
Yeah that's a good point! I'll probably refactor that in a separate PR

Reply:
Maybe just rename the variable from `successExplanation` to `resultExplanation`?

Reply:
I also want to make these changes across the resolver codebase in addition to the workflow definition

I'll get Openhands to do this for me in a separate PR

Reply:
I agree it makes sense to do the full renaming, I suspected it's in more places than just this (I must have seen it elsewhere in passing).

This isn't solved though, so I reopened it. 😅

Reply:
Ah right, thanks a bunch!